dist/container.js

```diff
@@ -2560,21 +2589,14 @@
 		 * Reference to the pause application button
 		 * @property {HTMLElement} pauseButton
 		 */
-		this.pauseButton = document.querySelectorAll(this.options.pauseButton);
+		this.pauseButton = document.querySelector(this.options.pauseButton);
```

Comment:
This change from QuerySelectorAll to QuerySelector breaks the container in Springroll Connect because there are two pause buttons

Reply:
Ah, nice catch! I'll go fix that.